### ISSUE TITLE
Support for including specific node_modules/*.scss files

### ISSUE THREAD

**Issue by dsod** [gencer/SCSS-Everywhere#58]
Hi, and thank you for a great extension!

I would like to be able to install bootstrap or other CSS frameworks through NPM, and add the framework into my project through SCSS @import's. Since (from what i understand) all files in node_modules folder are exceluded (hard coded?), the includeGlobPattern setting accepts a single string and @import statements are not supported, this is not possible.

I suggest one of the following solutions:

- If a @import statement in a SCSS file refers to a file that is currently not parsed, then parse said file
- Change the includeGlobPattern setting to accept an array of strings and parse all of them (so we can include a specific file in node_modules/...)
... In spite of the exclusion of node_modules.

// dsod

**Comment by gencer** [comment]
Hi @dsod. This could be done. However, I have to look for SCSS side to how should I parse this `@import` statement`first. Otherwise, we will go with your second option.

**Comment by kanlukasz** [comment]
Yes, that would be a very useful feature. I am using the Bootstrap framework and was also looking for a way to read classes from specific node_modules files

**Comment by gencer** [comment]
> Yes, that would be a very useful feature. I am using the **Bootstrap** framework and was also looking for a way to read classes from specific `node_modules` files

You can remotely load `css` files for bootstrap. This will give you the same autocompleted css values as scss gives.

Importing scss from another scss is still under review.
